app/screens/timezone/select_timezone/select_timezone.js

```diff
@@ -0,0 +1,199 @@
+import {ListTypes} from 'app/constants';
+import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
+
+import {getTimezoneRegion} from 'app/utils/timezone';
+
+const ITEM_HEIGHT = 45;
+const VIEWABILITY_CONFIG = ListTypes.VISIBILITY_CONFIG_DEFAULTS;
+
+export default class Timezone extends PureComponent {
+    static propTypes = {
+        selectedTimezone: PropTypes.string.isRequired,
+        initialScrollIndex: PropTypes.number.isRequired,
+        timezones: PropTypes.array.isRequired,
+        navigator: PropTypes.object,
+        onBack: PropTypes.func.isRequired,
+        theme: PropTypes.object.isRequired
+    };
+
+    static contextTypes = {
+        intl: intlShape
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            value: '',
+            timezones: props.timezones
+        };
+    }
+
+    filteredTimezones = (timezonePrefix) => {
+        if (timezonePrefix.length === 0) {
+            return this.state.timezones;
+        }
+
+        const lowerCasePrefix = timezonePrefix.toLowerCase();
+
+        return this.state.timezones.filter((t) => (
+            getTimezoneRegion(t).toLowerCase().indexOf(lowerCasePrefix) >= 0 ||
+            t.toLowerCase().indexOf(lowerCasePrefix) >= 0
+        ));
+    };
+
+    selectTimezone = (timezone) => {
+        this.props.onBack(timezone);
+        this.props.navigator.pop();
+    };
+
+    handleTextChanged = (value) => {
+        this.setState({value});
+    };
+
+    attachList = (l) => {
+        this.list = l;
+    };
+
+    keyExtractor = (item) => item;
+
+    getItemLayout = (data, index) => ({
+        length: ITEM_HEIGHT,
+        offset: ITEM_HEIGHT * index,
+        index
+    });
+
+    renderItem = ({item: timezone}) => {
+        const {theme, selectedTimezone} = this.props;
+        const style = getStyleSheet(theme);
+
+        const selected = timezone === selectedTimezone && (
+            <CheckMark
+                width={12}
+                height={12}
+                color={theme.linkColor}
+            />
+        );
+
+        return (
+            <TouchableOpacity
+                style={style.itemContainer}
+                key={timezone}
+                onPress={() => this.selectTimezone(timezone)}
+            >
+                <View style={style.item}>
+                    <Text style={style.itemText}>
+                        {timezone}
+                    </Text>
+                </View>
+                {selected}
+            </TouchableOpacity>
+        );
+    };
+
+    render() {
+        const {theme, initialScrollIndex} = this.props;
+        const {value} = this.state;
+        const {intl} = this.context;
+        const style = getStyleSheet(theme);
+
+        const searchBarInput = {
+            backgroundColor: changeOpacity(theme.sidebarHeaderTextColor, 0.2),
+            color: theme.sidebarHeaderTextColor,
+            fontSize: 15,
+            lineHeight: 66
```

Comment:
Might check to see if this is needed. I had to remove this to get text to show correctly on Android.

Reply:
Yeh looks like that's an issue with the dev branch.  Removing it did the trick